common/src/main/java/io/github/axolotlclient/modules/hypixel/HypixelAbstractionLayer.java

```diff
@@ -62,6 +64,13 @@ public static boolean hasValidAPIKey() {
 		return validApiKey;
 	}
 
+	public static JsonElement getPlayerProperty(String uuid, String stat) {
+		if (loadPlayerDataIfAbsent(uuid)) {
+			return getPlayer(uuid).getProperty(stat);
+		}
+		return new JsonObject();
```

Comment:
Considering the single place you use this has a null check, why not return null or an empty optional?

Reply:
done!